### PR TITLE
Adding per service healthchecks

### PR DESCRIPTION
**Pull request by bastosmichael**
Add BGS healthchecks per service in BGS gem per ticket https://vajira.max.gov/browse/API-1395

### CODE REVIEW

File: lib/bgs/base.rb [hunk 1]
```diff
@@ -155,6 +155,12 @@ def client
       @client ||= Savon.client(options)
     end
 
+    def healtcheck?
+      request(wsdl) ? true : false
```
Comment (edmangimelli) [inline lib/bgs/base.rb:159]: `wsdl` returns a URL, but `request` expects the first arg to be a `method`? I guess `method` doesn't mean HTTP verb here, right?

Comment (bastosmichael) [reply lib/bgs/base.rb:159]: From the way I read the code "method" appears to be the url as it's using in client.call(method) below but I'll let Charlie correct me if I'm wrong.

Comment (bastosmichael) [reply lib/bgs/base.rb]: Changed

File: bgs.gemspec [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   gem.name          = 'bgs_ext'
-  gem.version       = '0.14.1'
+  gem.version       = '0.14.2'
   gem.summary       = 'Thin wrapper on top of savon to talk with BGS'
   gem.description   = 'Thin wrapper on top of savon to talk with BGS for external BGS consumers'
   gem.license       = 'CC0' # This work is a work of the US Federal Government,
   #               This work is Public Domain in the USA, and CC0 Internationally
 
-  gem.authors       = 'Charley Stran'
+  gem.authors       = 'Charley Stran, Ed Mangimelli & Michael Bastos'
```
Comment (edmangimelli) [inline bgs.gemspec:9]: including me is pretty generous. lol

**Pull request merged**